gwsurrogate/new/nodeFunction.py

```diff
@@ -28,7 +28,7 @@
 THE SOFTWARE.
 """
 
-if __package__ is "" or "None": # py2 and py3 compatible
+if __package__ == "" or "None": # py2 and py3 compatible
   print("setting __package__ to gwsurrogate.new so relative imports work")
   __package__="gwsurrogate.new"
```

Comment:
python operator precedence says this is the same as testing the truth value of the string `"None"`, which is `True`, so the block is always executed. These three lines can just be replaced with the last line (minus the indentation)